README.md

```diff
@@ -24,16 +24,16 @@ Una vez en el [sitio](https://tinchou.github.io/organizador-fiuba/), automática
 
 ### Reportando, sugiriendo
 
-- Se pueden reportar horarios incorrectos [acá](https://github.com/tinchou/organizador-fiuba/issues/new?title=[CARRERAS][CÓDIGO][CURSO]&labels=horarios).
-- Se pueden dejar sugerencias [acá](https://github.com/tinchou/organizador-fiuba/issues/new?labels=sugerencias).
+- Se pueden reportar horarios incorrectos [acá](https://github.com/lugfi/organizador-fiuba/issues/new?title=[CARRERAS][CÓDIGO][CURSO]&labels=horarios).
+- Se pueden dejar sugerencias [acá](https://github.com/lugfi/organizador-fiuba/issues/new?labels=sugerencias).
 
 ### Programando
 
 1. Hacer un fork del repositorio
-1. Hacer las modificaciones necesarias en la rama `gh-pages`, la cual automáticamente levanta el [sitio](https://tinchou.github.io/organizador-fiuba/)
+1. Hacer las modificaciones necesarias en la rama `gh-pages`, la cual automáticamente levanta el [sitio](https://lugfi.github.io/organizador-fiuba/)
 1. Crear un pull request para integrar los cambios en este repositorio
 
 # Creditos
 
 - Franco Spada es el autor original del organizador
-- [Martín Coll](https://github.com/tinchou) mantiene este repositorio y la versión web
+- [Martín Coll](https://github.com/lugfi) mantiene este repositorio y la versión web
```

Comment:
Jaja, suelo hacer lio con reemplazar.
Lo de los créditos en particular no me parece mal

El 03/08/2016 09:49, "Martín Coll" <notifications@github.com> escribió:

> In README.md
> <https://github.com/lugfi/organizador-fiuba/pull/11#discussion_r73326703>:
>
> >  1. Crear un pull request para integrar los cambios en este repositorio
> >
> >  # Creditos
> >
> >  - Franco Spada es el autor original del organizador
> > -- [Martín Coll](https://github.com/tinchou) mantiene este repositorio y la versión web
> > +- [Martín Coll](https://github.com/lugfi) mantiene este repositorio y la versión web
>
> Esto no había que cambiarlo, pero se me puede sacar los créditos de última
> :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lugfi/organizador-fiuba/pull/11/files/ec7acdeaba65fc8842a3eb0fd86ab7f7c766abef..59f2552f3f979a7cdd2f2a39444ae5dca1a0f9c3#r73326703>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGsHApB7ud0i5JLZ4hFMUH_2lJ7V4flhks5qcIOpgaJpZM4JawH->
> .
>